### PR TITLE
minor front-end changes to home and login pages

### PR DESCRIPTION
**Pull request by Zalymo**

### CODE REVIEW

**Comment by DJ-IRL** [comment]
the more specific you can be about your changes with each commit the easier they are to review

File: osCam/templates/core/home.html [hunk 1]
```diff
@@ -44,7 +44,7 @@ <h2>cameras</h2>
                         <!-- cameras -->
                         {% if user.is_authenticated %}
                         <div class="container-fluid">
-                        <img src="{% url 'feed' %}">
+                        <img src="{% url 'feed' %}" style="border: 1px solid #eee">
```
Comment (DJ-IRL) [inline osCam/templates/core/home.html:47]: there's some styling still floating around the html but our goal is all css in dedicated files under "static/____.css"

File: osCam/templates/user/login.html [hunk 2]
```diff
@@ -9,12 +9,18 @@
...
-            {{ login_form.as_table }}
+            <label>{{ form.username.label_tag }}</label>
+            {{ form.username }} <br>
+            <label>{{ form.password.label_tag }}</label>
+            {{ form.password }} <br>
             <tr><td>
               <input type="submit" class="btn btn-primary" value = "Login"/>
             </td></tr>
-            <tr><td>Click <a href="/join">here</a> to register.</td></tr>
+            <tr><td style="padding-top: 25px">Don't have an account? <a href="/join" class="btn btn-secondary">Register</a></td></tr>
```
Comment (DJ-IRL) [inline osCam/templates/user/login.html:23]: i like "register" over "join." makes more sense. again styling in another file tho.

**Review by DJ-IRL — CHANGES_REQUESTED** [review]
just the move the css and it looks good to me.

File: osCam/templates/core/home.html (hunk 1 above)
Comment (Zalymo) [reply osCam/templates/core/home.html:47]: okay, I'll make a new file for the css

**Pull request merged**